src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,93 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav {
+  background-color: #fff;
+  height: 60px;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+}
+
+.nav__link {
+  margin-left: 20px;
+}
+
+.link {
+  display: flex;
+  position: relative;
+  align-items: center;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #060b35;
+  width: 100%;
+  font-weight: 500;
+  font-style: normal;
```

Comment:
normal is the default value for font-style you can remove this, fix in all places